src/EnergyPlus/ChillerExhaustAbsorption.cc

```diff
@@ -1888,7 +1888,13 @@ namespace EnergyPlus::ChillerExhaustAbsorption {
             }
 
             // Calculate operating part load ratio for cooling
-            lHeatPartLoadRatio = lHeatingLoad / lAvailableHeatingCapacity;
+            if (lAvailableHeatingCapacity <= 0.0) {
+                lAvailableHeatingCapacity = 0.0;
+                lHeatPartLoadRatio = 0.0;
+            } else
+            {
```

Comment:
Straightforward change to protect against divide by zero here and in the next chiller.
Minor formatting issue here, but that will get cleaned up with a global clang format sweep, so not holding this up for that.